R23.7 notes (#420)
stackabletech/documentation: modules/ROOT/pages/release_notes.adoc

The Stackable platform consists of multiple operators that work together.
Periodically a platform release is made, including all components of the platform at a specific version.

== Release 23.7

This release introduces the specification of resource quotas and Pod-overrides and updates the product versions supported by SDP.

=== New / extended platform features
The following new major platform features were added:

Resource Quotas::

> Explicit resources are now applied to all containers, both operators and products. This prevents pre-specified resource limits from being violated. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.

Reviewer: ```suggestion
Explicit resources are now applied to all containers, for both operators and products. This allows running the Stackable data platform on Kubernetes clusters with a ResourceQuota or LimitRange set. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.
```

Pod Overrides::

> It is now possible to add custom settings which specify elements of a pod template (service, stateteful set etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. See https://github.com/stackabletech/issues/issues/346[this issue] for more details.

Reviewer: ```suggestion
It is now possible to add custom settings which specify elements of a pod template (service, stateteful set etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. Have a look at https://docs.stackable.tech/home/stable/concepts/overrides.html#_pod_overrides[the documenation] for more details.
```

The Stackable platform consists of multiple operators that work together.
Periodically a platform release is made, including all components of the platform at a specific version.

== Release 23.7

This release introduces the specification of resource quotas and Pod-overrides and updates the product versions supported by SDP.

> === New / extended platform features
> The following new major platform features were added:

Reviewer 2: ```suggestion
=== New / extended platform features

The following new major platform features were added:
```

Resource Quotas::

Explicit resources are now applied to all containers, for both operators and products. This allows running the Stackable data platform on Kubernetes clusters with a ResourceQuota or LimitRange set. Where these are not specified directly, defaults will be used. See https://github.com/stackabletech/issues/issues/368[this issue] for more information.

> Pod Overrides::

Reviewer 2: Sometimes Pod overrides or pod-overrides, lets keep this consistent?

Author: Done. Changed to `pod overrides` as we use `configuration overrides` (lowercase) elsewhere.

> It is now possible to add custom settings which specify elements of a pod template (Service, StatetefulSet etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. This provides the user with a possibility for specifying any property that can be set on a regular Kubernetes Pod, but which is not directly exposed via the Stackable custom resource definition. Have a look at xref:concepts:overrides.adoc[the documentation] for more details.

Reviewer 2: ```suggestion
It is now possible to add custom settings which specify elements of a pod template (Service, StatefulSet etc.) on roles or rolegroups, which the operator then merges with the objects it writes before actually applying them. This provides the user with a possibility for specifying any property that can be set on a regular Kubernetes Pod, but which is not directly exposed via the Stackable custom resource definition. Have a look at xref:concepts:overrides.adoc[the documentation] for more details.
```
